src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements DrawFigure, AreaCalculator {
```

Comment:
if you implement DrawFigure and AreaCalculator in Figure class you don't have to implement them in every separate figure class because they will be inherited